File: DayTwo/Java/LevelTwo/TaskFour.java
import java.util.Scanner;

public class TaskFour{
    public static void main(String... args){
    
    Scanner input = new Scanner(System.in);
    
    System.out.print("Enter the first number: ");
    int numberOne = input.nextInt();
        
    System.out.print("Enter the second number: ");
    int numberTwo = input.nextInt();
    
    int largest = numberOne;
    
    if(numberTwo > largest){
        largest = numberTwo;    
    }
    
    System.out.print("The largest number = " + largest);

    }
}

File: DayTwo/JavaScript/LevelTwo/TaskFour.js
const prompt = require("prompt-sync")();

let numberOne = Number(prompt("Enter the first number: "))
let numberTwo = Number(prompt("Enter the second number: "))

let largest = numberOne;

    if(numberTwo > largest){
       largest = numberTwo;    
    }
    console.log(`The largest number = ${largest}`);

